spec.html

```diff
@@ -32395,6 +32529,115 @@ <h1>Expanded Years</h1>
+            TemporalSign Hour HourSubcomponents[~Extended]
+
+          TemporalSign :::
+            ASCIISign
+            U+2212
+
+          ASCIISign ::: one of
+            `+` `-`
+
+          Hour :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `20`
+            `21`
+            `22`
+            `23`
+
+          HourSubcomponents[Extended] :::
+            TimeSeparator[?Extended] MinuteSecond
+            TimeSeparator[?Extended] MinuteSecond TimeSeparator[?Extended] MinuteSecond TemporalDecimalFraction?
+
+          TimeSeparator[Extended] :::
+            [+Extended] `:`
+            [~Extended] [empty]
+
+          MinuteSecond :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `2` DecimalDigit
+            `3` DecimalDigit
+            `4` DecimalDigit
+            `5` DecimalDigit
+
+          TemporalDecimalFraction :::
+            TemporalDecimalSeparator DecimalDigit DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit?
+
+          TemporalDecimalSeparator ::: one of
+            `.` `,`
+        </emu-grammar>
+
+        <emu-clause id="sec-istimezoneoffsetstring" type="abstract operation">
+          <h1>
+            IsTimeZoneOffsetString (
+              _string_: a String,
+            ): a Boolean
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>The return value indicates whether _string_ conforms to the grammar given by |UTCOffset|.</dd>
+          </dl>
+          <emu-alg>
+            1. Let _parseResult_ be ParseText(StringToCodePoints(_offsetString_), |UTCOffset|).
+            1. If _parseResult_ is a List of errors, return *false*.
+            1. Return *true*.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-parsetimezoneoffsetstring" type="abstract operation">
+          <h1>
+            ParseTimeZoneOffsetString (
+              _offsetString_: a String,
+            ): an integer
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>The return value is the UTC offset, as a number of nanoseconds, that corresponds to the String _offsetString_.</dd>
+          </dl>
+          <emu-alg>
+            1. Let _parseResult_ be ParseText(StringToCodePoints(_offsetString_), |UTCOffset|).
+            1. Assert: _parseResult_ is not a List of errors.
+            1. Let each of _parsedSign_, _parsedHours_, _parsedMinutes_, _parsedSeconds_, and _parsedFraction_ be the source text matched by the |TemporalSign|, |Hour|, first |MinuteSecond|, second |MinuteSecond|, and |TemporalDecimalFraction| Parse Node contained within _parseResult_, or an empty sequence of code points if not present.
+            1. Assert: _parsedSign_ is not empty.
+            1. If _parsedSign_ is the single code point 0x002D (HYPHEN-MINUS) or 0x2212 (MINUS SIGN), then
```

Comment:
Code points use the U+XXXX notation and code units use the 0xXXXX notation. You probably wanted 

```suggestion
            1. If _parsedSign_ is U+002D (HYPHEN-MINUS) or U+2212 (MINUS SIGN), then
```